systems/node/notify/internal/server/notify.go

```diff
@@ -215,7 +215,7 @@ func add(nodeId, severity, nType, serviceName string, details []byte, nStatus ui
 		return err
 	}
 
-	route := baseRoutingKey.SetAction("store").SetObject("notification").MustBuild()
+	route := baseRoutingKey.SetAction("store").SetObject("node").MustBuild()
```

Comment:
We are storing a **notification**, not a "node"

Reply:
We discussed last time and agree to change that to node as the key will have alreay notify which means we don't need notification again.node.notify.node.store .

Reply:
Not sure if I was part of this discussion since i can't remember this specific point, still, I disagree for this naming because of the following:

1. **It is misleading:**  A good event name should be meaningful and not misleading especially without knowing the context, eg the event name should tell you exactly what is happening even if you do not know the context.  For example, if you randomly ask a group developers who do not know anything about the ukama repository or code base and ask them what they thinks this event  `"event.cloud.local.{{ .Org}}.node.notify.node.store"`  is about, the vast majority will tell you this is an event about some node being stored, which is false, misleading and a clear indication that this naming is wrong. 
2. **It violates our routing key naming scheme:**  Our routing key specs looks like `event.cloud.<scope>.org-name>.<system-name>.<service-name>.<object>.<action>` . The object is the thing that is being manipulated by the action (or the thing that is causing the action) and here the object is notification (not node)  because this event is happening because we are storing one notification row inside the notification DB, eg. DB.Notifications.Store on gorm (Remember, the DB is called notification, because it holds a bunch of notifications, and we are storing one of these notification). It is not a DB of nodes from which we are storing one node item, so calling  `"event.cloud.local.{{ .Org}}.node.notify.node.store"` an event triggered by a DB.Notifications.Store gorm operation is telling a false story.  The only thing that can make `"event.cloud.local.{{ .Org}}.node.notify.node.store"` not wrong is if we change the name of DB from **Notifications** to **Nodes**. Then and only then, the naming will be consistent, even if it won't make any sense to call a DB of notifications as DB.Nodes
3. **You are conflicting Notify and Notifications**.  Notify is the name of the service. Notification is one group of ojects manipulated by the Notify service. That's why we have a Notification DB inside that service. But this service can have many other DBs. Remember these events are about what happening inside the notify service itself, so if an event inside the notify service is generated because we are storing one notification item inside the Notification DB from the notify service, then the associated event should be named `"event.cloud.local.{{ .Org}}.node.notify.notification.store"` just for the sake of being correct and consistent.
4. **And I can go on and on and on ...** on why this naming is violating our own design principles.  Now, I still don't remember agreeing on this naming in a previous discussion but we can yet have a discussion.

Reply:
Routing key updated @delvatt